src/main/java/org/mule/tools/apikit/model/APIFactory.java

```diff
@@ -94,9 +102,10 @@ private String getNextPort(List<HttpListenerConfig> httpListenerConfigs) {
   }
 
   private HttpListenerConnection buildDefaultHttpListenerConnection(String port) {
+    boolean hasProperties = scaffoldingAccessories != null && scaffoldingAccessories.getProperties() != null;
     return new HttpListenerConnection.Builder()
-        .setHost(DEFAULT_HOST)
-        .setPort(port)
+        .setHost(hasProperties ? DEFAULT_HOST_PLACEHOLDER : DEFAULT_HOST)
```

Comment:
I'm wondering what happens if there are more than one api... It seems that all of them will end up with the placeholder and the same port set by the properties file. right?
I'm just trying to understand the code behind the calculated "next port" passed as argument here.